--- gitea/charts/postgresql/values.yaml	2023-11-27 14:02:43.000000000 -0700
+++ ../values.yaml.postgresql	2024-01-09 11:08:45.211532244 -0700
@@ -455,7 +455,7 @@
   ##
   podSecurityContext:
     enabled: true
-    fsGroup: 1001
+      #fsGroup: 1001
   ## Container Security Context
   ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
   ## @param primary.containerSecurityContext.enabled Enable container security context
@@ -468,7 +468,7 @@
   ##
   containerSecurityContext:
     enabled: true
-    runAsUser: 1001
+      #runAsUser: 1001
     runAsGroup: 0
     runAsNonRoot: true
     allowPrivilegeEscalation: false
@@ -818,7 +818,7 @@
   ##
   podSecurityContext:
     enabled: true
-    fsGroup: 1001
+      #fsGroup: 1001
   ## Container Security Context
   ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
   ## @param readReplicas.containerSecurityContext.enabled Enable container security context
@@ -831,7 +831,7 @@
   ##
   containerSecurityContext:
     enabled: true
-    runAsUser: 1001
+      #runAsUser: 1001
     runAsGroup: 0
     runAsNonRoot: true
     allowPrivilegeEscalation: false
@@ -1103,7 +1103,7 @@
     ## @param backup.cronjob.podSecurityContext.fsGroup Group ID for the CronJob
     podSecurityContext:
       enabled: true
-      fsGroup: 1001
+        #fsGroup: 1001
     ## backup container's Security Context
     ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container
     ## @param backup.cronjob.containerSecurityContext.runAsUser User ID for the backup container
@@ -1114,7 +1114,7 @@
     ## @param backup.cronjob.containerSecurityContext.seccompProfile.type Set backup container's Security Context seccompProfile type
     ## @param backup.cronjob.containerSecurityContext.capabilities.drop Set backup container's Security Context capabilities to drop
     containerSecurityContext:
-      runAsUser: 1001
+      #runAsUser: 1001
       runAsGroup: 0
       runAsNonRoot: true
       allowPrivilegeEscalation: false
@@ -1323,7 +1323,7 @@
   ## @param volumePermissions.containerSecurityContext.seccompProfile.type seccompProfile.type for the init container
   ##
   containerSecurityContext:
-    runAsUser: 0
+    #runAsUser: 0
     runAsGroup: 0
     runAsNonRoot: false
     seccompProfile:
@@ -1441,7 +1441,7 @@
   ##
   containerSecurityContext:
     enabled: true
-    runAsUser: 1001
+      #runAsUser: 1001
     runAsGroup: 0
     runAsNonRoot: true
     allowPrivilegeEscalation: false
@@ -1598,4 +1598,4 @@
     ##       description: replication for {{ include "common.names.fullname" . }} PostgreSQL is lagging by {{ "{{ $value }}" }} hour(s).
     ##       summary: PostgreSQL replication is lagging by {{ "{{ $value }}" }} hour(s).
     ##
-    rules: []
+    rules: []
\ No newline at end of file
